### ISSUE TITLE
Two controller of same type control same controller

### ISSUE THREAD

**Issue by jabroany** [jakibaki/hid-mitm#15]
Using the 1st release (beta) where only 1 controller can be emulated, if I plug two of the same controller types (ie 2 xbox360 controllers) they both will control the switch emulated controller.

Did it using parsec, where the connected guys both's controllers become x360.

**Comment by Jackall4BDN** [comment]
Yeah, i would second that, simply a list where to choose which controller sowkrs for the program or a list of checkmarks would suffice, so i can physically disconnect other controlelrs from the pc client
